app/experimenter/nimbus-ui/src/hooks/useExperiment.tsx

```diff
@@ -54,7 +54,11 @@ export function useExperiment(slug: string) {
     fieldPageMap[page].some((field) => missingFields.includes(field)),
   );
   const isMissingField = (fieldName: string) =>
-    missingFields.includes(fieldName);
+    missingFields.includes(fieldName) &&
+    // This is a bit hacky, but we only want to visually display missing field
+    // errors when you click the links under the "Missing details" section, not
+    // from a regular sidebar navigation link
+    window.location.search.includes("show-errors");
```

Comment:
Also open to suggestions on this approach. We want to show the ( ! ) field icons only when links from the "Missing Details" section of the sidebar are clicked, and not from the regular sidebar links.

Reply:
Seems like the more "proper" way would involve some prop-drilling or context shenanigans involving the router that we might want to defer for now?

Reply:
Works for me: https://jira.mozilla.com/browse/EXP-651